main.py

```diff
@@ -93,6 +105,33 @@ def draw_overmap(screen, position):
         pg.draw.rect(screen, OVERMAP_BG, mask)
 
 
+def change_color_background(index, color):
+    """Permet de changer la couleur du fond d'écran à partir du menu"""
+    global BACKGROUND_COLOR
+    BACKGROUND_COLOR = COLORS[color]
+
+
+def change_color_blop(index, color):
+    """Permet de changer la couleur du blop à partir du menu"""
+    global COLOR_BLOP
+    COLOR_BLOP = COLORS[color]
+
+
+def get_player_name(name):
+    """Permet de récupérer le nom du joueur à partir du menu"""
+    global PLAYER_NAME
+    PLAYER_NAME = name
+
+
+def PAUSE_MENU(pauseMenu, screen):
```

Comment:
Nom de function toujours en minuscue (ie camel_case)

```suggestion
def display_pause_menu(pauseMenu, screen):
```